app/code/core/Mage/Adminhtml/Block/Catalog/Product/Edit/Tab/Alerts/Price.php

```diff
@@ -30,7 +30,7 @@ public function __construct()
         $this->setDefaultDir('desc');
         $this->setUseAjax(true);
         $this->setFilterVisibility(false);
-        $this->setEmptyText(Mage::helper('catalog')->__('There are no customers for this alert'));
+        $this->setEmptyText(Mage::helper('catalog')->__('There are no customers for this alert.'));
```

Comment:
I think I wouldn't break the translation packs just to add the dot :-(

Reply:
The translation already exists.

Reply:
true, didn't notice, sorry!